src/recording/macromanager.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include <QObject>
+
+#include "engine/enginemaster.h"
+#include "mixer/playermanager.h"
+#include "preferences/usersettings.h"
+#include "recording/recordingmanagerbase.h"
+
+#define MAX_MACRO_SIZE 1000
```

Comment:
constexpr size_t kMaxMaroSize = 1000;

And put this into the anonymous namespace

Reply:
Ah okay, wasn't quite sure how to put it, thx